src/ukify/ukify.py

```diff
@@ -1726,6 +1733,16 @@ def config_example(self) -> tuple[Optional[str], Optional[str], Optional[str]]:
         config_key='UKI/Initrd',
         config_push=ConfigItem.config_list_prepend,
     ),
+    ConfigItem(
+        '--efifw',
+        metavar='PATH',
+        type=Path,
+        action='append',
+        default=[],
+        help='firmware file [.efifw section]',
+        config_key='UKI/Firmware',
+        config_push=ConfigItem.config_list_prepend,
+    ),
```

Comment:
btw, I think it would make sense if ukify would actually put together the header automatically. I think we should just use the basename of the provided path here as fwid.

Reply:
OK not sure what changes you want here. Do you want to make `metavar` a `DIR` type? How do you envision we process that DIR?

Reply:
I suppose path to an xml file with `fwid` string and a patch to the efifw firmware binary would also work?

Reply:
i was thinking that if you specify `--efifw=/my/firmware/repository/myboard-4711.quux` we'd automatically synthesize an .efifw section from it that uses "myboard-4711.quux" is fwid, with the file contents as payload.

yes, the .efifw header is extensible and could cover more fields, but for now we don't need that, and i don#t think we need to cover that with ukify for now. when the time comes where we actually want to set more fields, we can still add `--efifw-description=` or so which defines where to read the other fields from. But let's not complicate things with that for now.

sorry, but XML is not ok for new stuff. It's a legacy format in our eyes.

Reply:
But xml is easily extensible with other attributes in the future but your approach seems brittle?

Reply:
I guess the ukify changes, that `.efifw` section building stuff needs to be part of this PR?